lib/osf-components/addon/components/search-result-card/user-secondary-metadata/template.hbs

```diff
@@ -0,0 +1,30 @@
+<CpPanel class={{@wrapperClass}} id={{@wrapperId}} @open={{@isOpen}} as |panel|>
+    <panel.body {{did-update this.onToggle @isOpen}}>
+        <hr>
+
+        {{#if this.getOsfUserModel.isRunning}}
+            <LoadingIndicator @dark={{true}}/>
+        {{else if this.getOsfUserModel.isError}}
+            {{t 'osf-components.search-result-card.fetch_user_error'}}
+        {{else if (not this.user)}}
+            {{t 'osf-components.search-result-card.no_user'}}
+        {{else}}
+            <dl>
+                {{#if this.user.employment.length}}
+                    <dt>{{t 'osf-components.search-result-card.employment'}}</dt>
+                    <dd>{{this.user.employment.[0].institution}}</dd>
+                {{/if}}
+                {{#if this.user.education.length}}
+                    <dt>{{t 'osf-components.search-result-card.education'}}</dt>
+                    <dd>{{this.user.education.[0].institution}}</dd>
```

Comment:
Are we only doing the latest education/employment, rather than all of them or the one marked 'current'?

Reply:
Yea, I spoke with Eric, and he wants just the latest one, or more accurately, whichever one the users put at the top of the list

Reply:
Yea, I spoke with Eric, and he wants just the latest one, or more accurately, whichever one the users put at the top of the list